lib/cmds/content-type_cmds/list.js

```diff
@@ -17,25 +18,33 @@ export const builder = (yargs) => {
   return yargs
     .option('space-id', { alias: 's', type: 'string', describe: 'Space id' })
     .option('management-token', { alias: 'mt', type: 'string', describe: 'Contentful management API token' })
+    .option('environment-id', {
+      alias: 'e',
+      type: 'string',
+      describe: 'Environment ID you want to interact with. Defaults to the current active environment.'
+    })
     .epilog('Copyright 2018 Contentful, this is a BETA release')
 }
 
 async function ctList (argv) {
   await assertLoggedIn(argv)
   await assertSpaceIdProvided(argv)
 
-  const { cmaToken, activeSpaceId } = await getContext()
+  const { cmaToken, activeSpaceId, activeEnvironmentId } = await getContext()
   const managementToken = argv.managementToken || cmaToken
   const spaceId = argv.spaceId || activeSpaceId
+  const environmentId = argv.environmentId || activeEnvironmentId
```

Comment:
What do you think about setting the `environmentId` to `master` when the user does not pass anything and there is no activeEnvironmentId?

Reply:
Yeah, thought to do this but held back because I didn't know how our conversations regarding implementing environment branching/recovering with aliases was going to affect this. I'll set "master" as a fallback here for now and we can maybe adjust accordingly in the future.